src/components/OneToDo/OneToDo.jsx

```diff
@@ -5,35 +5,23 @@ import SaveIcon from "@mui/icons-material/Save";
 import React, { useRef } from "react";
 import { useState } from "react";
 import "./OneToDo.css";
-import { useDispatch, useSelector } from "react-redux";
-import {
-  editToDo as reduxEditToDo,
-  deleteToDo as reduxDeleteToDo,
-} from "../../redux/toDoSlice/toDoSlice";
+import connectOneToDo from "../containers/OneToDoContainer";
```

Comment:
why we are still importing connectOneTodo here?

Reply:
it's a default export so I didn't change the naming, just updated the import location to the container file